lib/swagger/generator.rb

```diff
@@ -18,6 +26,18 @@ module SwaggerGenerator
     # http://files.slatestudio.com/gG30
     ALLOW_TYPES  = %w(Object String Integer Array Date Mongoid::Boolean Symbol)
 
+    def swagger_options(options)
```

Comment:
So the way this works is:
1. set custom values with `swagger_options`
2. if value is set it's used as is
3. if value is missing default value is set